src/pages/DataAccessRequestApplication.js

```diff
@@ -1139,4 +1139,4 @@ class DataAccessRequestApplication extends Component {
   }
 }
 
-export default DataAccessRequestApplication;
+export default DataAccessRequestApplication;
```

Comment:
minor nit - we don't need to remove this line break. In general, all files should end with a line break.